products/mlengine/terraform.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Model: !ruby/object:Overrides::Terraform::ResourceOverride
```

Comment:
Does the `regions` list always come back in the same order that was passed to it? If not, then it should be a set (override with `is_set: true`).

oh I guess it only supports one region for now according to your docs. In that case, let's add a `max_size` to it so we can catch bad configs at plan-time.

Reply:
Yeah, the API only supports one region now. I'm assuming it'll support many at some point in the future.